### ISSUE TITLE
Move bad exchanges to a separate list instead of removing them

### ISSUE THREAD

**Issue by xaur** [pluja/kycnot#30]
I noticed #24 and realized that the info about why each exchange was delisted (or was considered but rejected) is valuable and it would be good to have it more visible on the site, compared to being buried in commits or GitHub issues. It aggregates good investigative work that saves time.

Instead of just removing exchanges that don't qualify, I suggest to move them to a different area (e.g. "blacklist" or "bad"), with links to all known evidence about why they were moved there.

**Comment by pluja** [comment]
That's true! Good idea. For now I don't have any time for myself and nor any of my projects, so I think it will be at least some time until I can start working on this... At least 1 month if things don't change. So all contributors are really welcome and I will be reviewing this repo and issues as frequently as I can!

**Comment by serejandmyself** [comment]
+100 to this idea
